assets/js/base/utils/address.ts

```diff
@@ -100,3 +100,12 @@ export const emptyHiddenAddressFields = <
 
 	return newAddress;
 };
+
+/**
+ * Returns true if the address has a city and country.
+ */
+export const isAddressComplete = (
+	address: ShippingAddress | BillingAddress
+): boolean => {
+	return !! address.city && !! address.country;
+};
```

Comment:
While testing this PR, I noticed that I also had to provide the street name before the shipping costs appeared. Besides that, I ran a global search for this function, and it doesn't seem to be used anywhere. Can you confirm that?

Reply:
> Thanks for spotting that, I got rid of it! 🗑️

Thanks, @opr.

> What do you think of that, should we make it so only city/state/country/postcode are required?

I think, we should keep the street name as a required field. When I organised [WordCamp Ubud 2017](https://ubud.wordcamp.org/2017/), one of the organisers, who ordered swags from WordCamp Central, accidentally forgot to mention the street address. As a result of that, the swags never arrived. To prevent others from making the same mistake accidentally, I'd keep the street name field required.